src/projects/ccproject.vala

```diff
@@ -62,7 +62,12 @@ class Vls.CcProject : Project {
             if (cc == null)
                 throw new ProjectError.INTROSPECTION (@"JSON node is null. Bailing out!");
 
-            if (cc.command.length > 0 && cc.command[0].contains ("valac"))
+            if (cc.command.length == 0) {
+                warning ("CompileCommand has empty command");
```

Comment:
Include more context for this message, i.e.:

```vala
warning ("CC #%d has empty command list", i);
```

Reply:
updated